riscv-c-api.md

```diff
@@ -342,8 +358,22 @@ Each `TARGET-VERSION-ATTR-STRING` defines a distinguished version of the functio
 The syntax of `<TARGET-VERSION-ATTR-STRING>` describes below:
 
 ```
-TARGET-VERSION-ATTR-STRING := 'arch=' EXTENSIONS
-                            | 'default'
+TARGET-VERSION-ATTR-STRING := 'default'
```

Comment:
You have this same non-trivial BNF repeated twice.  Maybe replace the second with a back reference to the previous?  "is the same as described above for target_clones"?  (You can also link that text.)